### PR TITLE
fix: lint

### PR DESCRIPTION
**Pull request by jannyHou**
Quick fix for lint

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
+    80
+  ],
```
Comment (raymondfeng) [inline .vscode/settings.json]: I'm curious why the change is needed. `npm run lint` does not check this file, right?

Comment (jannyHou) [reply .vscode/settings.json]: @raymondfeng nope, I my vscode breaks it into multiple lines automatically when I save the file...

See discussion in https://github.com/strongloop/loopback-next/pull/2690/files#diff-5b46cfda0ddaed371f52aa24ea2b0b2a

Comment (dhmlau) [reply .vscode/settings.json]: I tried to save this file and VS code doesn't reformat for me. Just FYI.

Comment (jannyHou) [reply .vscode/settings.json]: ok seems other people don't have the problem. I can close the PR.

**Comment by jannyHou** [comment]
seems like only my local vscode complains. Closing it.
